openreview/openreview-web
# Feature/ React edit mode banner

Add an EditBanner component and switch group, invitation, and profile pages to use it.

Fixes #751 #864 and #822 

## Testing

- Verify that the issues are fixed
- Verify that the banner is displayed correctly on the profile page and that switching back and forth between edit mode and view mode works
- Verify that the banner is displayed correctly on the group page and that switching back and forth between edit mode, view, and info mode works
- Verify that the banner is displayed correctly on the invitation page and that switching back and forth between edit, view, and info mode works
